account/identity.go

```diff
@@ -21,6 +22,11 @@ import (
 const (
 	// KeycloakIDP is the name of the main Keycloak Identity Provider
 	KeycloakIDP string = "kc"
+
+	IdentityResourceTypeOrganization = "identity/organization"
+	IdentityResourceTypeTeam         = "identity/team"
+	IdentityResourceTypeGroup        = "identity/group"
+	IdentityResourceTypeUser         = "identity/user"
```

Comment:
Wondering if for consistency's sake we should name these `openshift.io/resource......` or is it because we want to name these differently ?

Reply:
This naming was intentional, I wanted to differentiate these resource types from the openshift.io resource types, as they are only handled internally by the auth service.